x-pack/platform/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -652,13 +652,54 @@ export default function ({ getService }: FtrProviderContext) {
         .set('kbn-xsrf', 'xxx')
         .expect(200);
 
+      // api key should still exist
       expect(
         queryResult.body.apiKeys.filter((apiKey: { id: string }) => {
           return apiKey.id === result.userScope?.apiKeyId;
         }).length
-      ).eql(0);
+      ).eql(1);
```

Comment:
Updated this test that previously checked whether the API key was deleted after a task was run. Now it checks that the API key is still there an an `api_key_to_invalidate` SO is created. Then it manually triggers the invalidate API key task and checks that the API key is deleted after that.